integration-test/src/main/java/com/sequenceiq/it/cloudbreak/dto/environment/EnvironmentTestDto.java

```diff
@@ -212,11 +214,15 @@ public EnvironmentTestDto withNetwork(String key) {
     }
 
     public EnvironmentTestDto withS3Guard(String tableName) {
-        AwsEnvironmentParameters awsEnvironmentParameters = new AwsEnvironmentParameters();
-        S3GuardRequestParameters s3GuardRequestParameters = new S3GuardRequestParameters();
-        s3GuardRequestParameters.setDynamoDbTableName(tableName);
-        awsEnvironmentParameters.setS3guard(s3GuardRequestParameters);
+        getCloudProvider().setS3Guard(this, tableName);
+        return this;
+    }
+
+    public EnvironmentTestDto withSingleResourceGroup(String resourceGroupUsage, String resourceGroupName) {
```

Comment:
I would rather call this withResourceGroupUsage because it can be used to set "multiple" model, too

Reply:
Thanks! I renamed it to withResourceGroup, without usage, as one parameter is called resourceGroupUsage.